packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -48,7 +48,7 @@ export async function fetchSnapshot(
     fetchFullSnapshot: boolean,
     logger: ITelemetryLogger,
     snapshotDownloader: (url: string, fetchOptions: {[index: string]: any}) => Promise<IOdspResponse<unknown>>,
-): Promise<ISnapshotContents> {
+): Promise<ISnapshotResponse> {
```

Comment:
I do not think we want to change that function - it does not deal with expiration, so it should keep returning ISnapshotContents, and callers should deal with transforming it to ISnapshotResponse